modules/ui/src/main/resources/lucuma-css/lucuma-ui.scss

```diff
@@ -27,13 +27,68 @@
   }
 }
 
-// For styling of buttons in primereact for different "compactness"
+// For styling of buttons in primereact for different "compactness" and "size"
 button.p-component.p-button {
   margin-right: .25em; // this is what the current css does for SUI. Might want to improve.
 
-  // Probably will need to 'specialize' for small and large buttons, and compact vs very compact
-  &.pl-compact,
-  &.pl-very-compact {
-    padding: .4em .5em;
+  &.pl-compact {
+    padding: 0.39285714em;
+  }
+
+  svg.svg-inline--fa {
+    width: 1.5em;
+  }
+
+  .p-button-icon-left {
+    margin-right: 0.2em;
+  }
+
+  .p-button-icon-right {
+    margin-left: 0.2em;
+  }
+
+  // default size is Medium
+  &.pl-mini {
+    font-size: 0.78571429rem;
+  }
+
+  &.pl-tiny {
+    font-size: 0.85714286rem;
+  }
+
+  &.p-button-sm,
+  &.pl-small {
+    font-size: 0.92857143rem;
+  }
+
+  &.p-button-lg,
+  &.pl-large {
+    font-size: 1.14285714rem;
+  }
+
+  // unfortunately, primereact uses `scaling factors` to scala the `lg` and `sm` component. It also 
+  // seems to apply these to padding, which we don't want. Change the scaling factors seemed inappropriate
+  // because it would affect many more componenents, so I had to copy the padding value for the medium button.
+  &.p-button-sm,
```

Comment:
This, and the 2 selectors above, allow primereact `Small`, `Normal` and `Large` sizes to be used instead of the new `Small`, `Medium` and `Large` CSS classes. But, this selector relies even more on knowing what the values we set in the primereact-designer are. If we commited to ignoring the primereact button sizes and just used the classes or the extension methods, we could get rid of this.